orderapi/src/main/java/fi/hel/verkkokauppa/order/listeners/DeadLetterQueueListener.java

```diff
@@ -111,7 +112,7 @@ private void sendNotificationToEmail(PaymentMessage paymentMessage) {
 
             msgJson.put("body", html);
         } catch (IOException | URISyntaxException e) {
-            throw new RuntimeException(e);
+            log.info("Failed to serialize email template: {}", e.toString());
```

Comment:
Do not delete the runtimeException throw?

Reply:
And use object mapper to write that exception like in default active mq error handler